roles/wp-cli/defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 phar_url: https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
 bin_path: /usr/bin/wp
+wp_cli_completion_url: https://raw.githubusercontent.com/wp-cli/wp-cli/master/utils/wp-completion.bash
```

Comment:
We internally debated about whether we should be using a nightly version at all. Because then there's a difference between dev/prod.